### ISSUE TITLE
Support `alert tag / edge type name and prop name`

### ISSUE THREAD

**Issue by GourdErwa** [vesoft-inc/nebula#4606]
Whether to support `alert tag / edge type name and prop name`.
In the real environment, schema changes frequently.

**Comment by VincentSleepless** [comment]
i want this feature !!!!  more flexable asset entity defiation !!!

**Comment by wey-gu** [comment]
You meant alter name, right?

CC @MuYiYong

**Comment by GourdErwa** [comment]
> You meant alter name, right?
> 
> CC @MuYiYong

Yes, change the name. In actual use, the tag and relationship design may change the initial name.
